### PR TITLE
wxGUI: update menu creation explanations

### PR DESCRIPTION
**Pull request by neteler**
- remove outdated Requirements and point instead to central file
- update some paths to tools
- manual menu creation -> automated menu creation

Suggestion: if no objections I'd turn this file into a MD file.

### CODE REVIEW

**Comment by neteler** [comment]
Cool, the "Title CI" works :)

**Comment by echoix** [comment]
Is it parsed for some docs or only for the developpers in the repo? If it is only for the repo, go ahead and change it to markdown, I can't even see a rich view that would show if the relative links to files work correctly or not. If it is intended to be ReST for sphinx docs, thats another thing..

**Comment by neteler** [comment]
> it is only for the repo

Yes, to my knowledge. Now changed to markdown in db1ef21.

File: gui/wxpython/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# GRASS GIS - wxPython Graphical User Interface (wxGUI)
+
+## 1 - REQUIREMENTS
+
+See `../../REQUIREMENTS.md`
```
Comment (echoix) [inline gui/wxpython/README.md:5]: Change as relative link? Something with at least the parentheses part like this, the label is up to choice.
```suggestion
See `[../../REQUIREMENTS.md](../../REQUIREMENTS.md)`
```

File: gui/wxpython/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+## 4 - DEBUGGING
+
+To enable GUI debug messages on given level set `WX_DEBUG` environment
+variable, e.g.
+
+`$ g.gisenv set="WX_DEBUG=3"`
+
+## 5 - CLI BASED DISPLAY USAGE
+
+Use command `d.mon` (shell script in `gui/scripts/` directory) to start map
```
Comment (echoix) [inline gui/wxpython/README.md:35]: A link?

File: gui/wxpython/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+You should be able to zoom & pan through the map, once the layers are
+displayed. You should be also able to store the display content as well as
+clear the display and start from scratch.
+
+## 6 TRANSLATION HANDLING
+
+Notes:
+
+- Help part of menu entries is coming from the module descriptions
+- The menu is generated by `./tools/build_modules_xml.py`
```
Comment (echoix) [inline gui/wxpython/README.md:58]: A link?

File: gui/wxpython/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+Update of module description strings in `./xml/menudata.xml`:
+
+- in a GRASS session, run `./tools/update_menudata.py`
+
+From this (updated) `./xml/menudata.xml`, the gettext strings are generated
+via Makefile and stored into the file `menustrings.py`.
+
+While generating the po files in `../../locale/po/` by `../../locale/Makefile`,
+all `.py` files are parsed and the strings are stored in
+`../../locale/po/grasswxpy_XX.po`.
```
Comment (echoix) [inline gui/wxpython/README.md:69]: Would links be useful here?

Comment (neteler) [reply gui/wxpython/README.md:69]: Honestly, I don't think so. It was a rather command line focused file as far as I understand it. 
But no strong opinion...

Comment (echoix) [reply gui/wxpython/README.md:69]: Me neither.. so let's leave it as is. Only the first one that referred to more content to read was important in my opinion.

**Review by echoix — APPROVED** [review]

**Pull request merged**